src/views/AddItem.jsx

```diff
@@ -1,3 +1,114 @@
-export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+import { useEffect, useState } from 'react';
+import { addItem } from '../api/firebase';
+
+const timeframeToDays = {
+	Soon: 7,
+	'Kind of Soon': 14,
+	'Not Soon': 30,
+};
+
+export function AddItem({ listId }) {
+	const [timeframe, setTimeframe] = useState('soon');
```

Comment:
Currently no default is being selected for timeframe and when you try to submit an item name without selecting the time frame, it says 'Added strawberries to your list' but then doesn't actually add because it's missing the timeframe.

Just testing it out it looks like the default is just not picking up correctly in the useState for timeframe.  Looks like it is just a little typo and changing the default on useState to 'Soon' to match the Soon key up top in timeframeToDays works and then adds it to the List correctly.